### DISCUSSION TITLE
How to create a partitioned table in Postgres?

### DISCUSSION THREAD

**Discussion by patrick-siewe-resourcely** [go-gorm/gorm#6501] [Q&A]
How can I create a partitioned Postgres table using GORM?

```
create table post(
ts timestamptz not null default now(),
title varchar(10)not null
) partition by RANGE (ts);
```

I found this [question on StackOverflow](https://stackoverflow.com/questions/71738187/is-there-a-way-to-achieve-postgres-table-partitioning-using-gorm-v2) and [this GitHub issue](https://github.com/go-gorm/gorm/issues/4668) but neither has any hint.

**Comment by garrysanders** [comment]
Bumping. As we're keen on this as well
